notebook-tiny/pyspark.ipynb

```diff
@@ -435,11 +435,19 @@
     "def trimFieldImpl(js):\n",
     "    try:\n",
     "        d = json.loads(js)\n",
-    "        return [d.get(f) for f in interesting_fields]\n",
```

Comment:
@tuxdna thanks for submitting a patch!  I think that the only way the `try` block in the original code can (realistically) fail is if `json.loads` fails (and it actually does on some of the fedmsg data) -- `d.get(f)` will always either return a value or `None` (see the `dict_get` implementation [here](http://svn.python.org/view/%2acheckout%2a/python/tags/r266/Objects/dictobject.c)) and thus has the same behavior as the suggested change but in fewer lines of code.  There are other ways we could fail (running out of heap/address space (!), etc) but it is hard to see how we could recover from these.